nvdaHelper/remote/COMProxyRegistration.cpp

```diff
@@ -140,8 +139,11 @@ COMProxyRegistration_t* registerCOMProxy(wchar_t* dllPath) {
 		for(unsigned short idx=0;idx<fileInfo.TableSize;++idx) {
 			IID iid=*(fileInfo.pStubVtblList[idx]->header.piid);
 			CLSID clsidBackup={0};
-			wstring_convert<codecvt_utf8_utf16<wchar_t>> converter;
-			wstring name=converter.from_bytes(fileInfo.pNamesArray[idx]);
+			int nameLength=MultiByteToWideChar(CP_UTF8,0,			fileInfo.pNamesArray[idx],-1,NULL,0);
+			wchar_t* wcharName=(wchar_t*)calloc(nameLength+1,sizeof(wchar_t));
+			MultiByteToWideChar(CP_UTF8,0,			fileInfo.pNamesArray[idx],-1,wcharName,nameLength);

```

Comment:
Please don't use calloc and free directly. Instead either use RAII and smart pointer objects to manage memory for you, or because you really want a string just access the memory of the string directly rather than constructing it from (and copying) a temporary buffer.

The [data method on basic_string will help with this](https://en.cppreference.com/w/cpp/string/basic_string/data)

See the following for more complete example, though note they aren't using the data function. https://stackoverflow.com/questions/46867436/fill-stdstring-directly-from-c-api/46867492#46867492